### ISSUE TITLE
Always getting "something went wrong" error on `auth0 branding texts update signup` command

### ISSUE THREAD

**Issue by maggie9595** [auth0/auth0-cli#515]
### Checklist

- [X] I have looked into the [README](https://github.com/auth0/auth0-cli#readme) and have not found a suitable solution or answer.
- [X] I have looked into the [documentation](https://auth0.github.io/auth0-cli/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/auth0-cli/issues) and have not found a suitable solution or answer.
- [X] I have upgraded to the [latest version](https://github.com/auth0/auth0-cli/releases/latest) of this project and the issue still persists.
- [X] I have searched the [Auth0 Community](https://community.auth0.com/c/sdks/5) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

The same `auth0 branding texts update signup` command that we are running for our tenants in version `0.11.7` seems to be erroring out persistently after we updated to version `0.12.0`, with this error:
```
!!     Uh oh. Something went wrong.
!!     If this problem keeps happening feel free to report an issue at
!!
!!     https://github.com/auth0/auth0-cli/issues/new/choose
```

Reinstalling did not seem to fix the error either.

### Expectation

`auth0 branding texts update signup` should not error out in the new version of the CLI.

### Reproduction

- Update to newest version (0.12.0) of the CLI
- Try to run `auth0 branding texts update signup` command on any tenant
- Get persistent `something went wrong` error

### Auth0 CLI version

0.12.0

**Comment by jdk2pq** [comment]
I'm having the same issues. Every prompt I've tried gives the above error except for `auth0 branding texts update login`.

**Comment by maggie9595** [comment]
> I'm having the same issues with any `auth0 branding texts update` command. Every prompt I've tried gives the above error, except for `auth0 branding texts update login`.

This is actually the case for me as well, just updated my issue!

**Comment by jameskyker** [comment]
Auth0's API is broken. We tried to issue the GETs [described here](https://auth0.com/docs/api/management/v2#!/Prompts/get_custom_text_by_language), and none of them seem to work except for login. The login prompt returns values, but signup, for instance, returns an empty object as well as every other we tried. Auth0 needs to fix their API, and then this client will probably work.

**Comment by jameskyker** [comment]
Never mind. We did some additional testing today, and the GETs aforementioned return an empty object when there is no customization. Once we customized a few fields via the API, the GET is returning data. Therefore, the API is not busted as I stated. The client is. Sorry for the distraction here.

**Comment by willvedd** [comment]
Hi folks, appreciate everyone's feedback and patience. I've confirmed that all invocations of the `auth0 branding texts update` with a non-`login` prompt type result in an error. It appears that there is some faulty logic when trying to seed default branding texts to the editor. 

I have [PR](https://github.com/auth0/auth0-cli/pull/584) up to fix this issue, however it'll be gated behind our `v1` branch. What this means is that this fix won't be released until our first major release. We're targeting the next few weeks for beta and GA in mid-February; it's something we're very excited about. We acknowledge the wait and appreciate everyone's patience until then. I will be providing further updates as these releases go out.

**Comment by smart** [comment]
I downloaded the 1.0 beta3 and tried this command and still got the invalid memory issue

./auth0 universal-login templates update
⣽panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x105404998]

goroutine 25 [running]:
github.com/auth0/auth0-cli/internal/cli.fetchBrandingSettingsOrUseDefaults({0x105b4fcd0?, 0x1400070e0f0}, 0x1400044e000)
	/home/runner/work/auth0-cli/auth0-cli/internal/cli/universal_login_templates.go:292 +0x1b8
github.com/auth0/auth0-cli/internal/cli.(*cli).fetchTemplateData.func4()
	/home/runner/work/auth0-cli/auth0-cli/internal/cli/universal_login_templates.go:223 +0x34
golang.org/x/sync/errgroup.(*Group).Go.func1()
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.1.0/errgroup/errgroup.go:75 +0x5c
created by golang.org/x/sync/errgroup.(*Group).Go
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.1.0/errgroup/errgroup.go:72 +0x9c

**Comment by willvedd** [comment]
Hi all, the fix for this issue (including @smart 's point) has been released with [v1.0.0](https://github.com/auth0/auth0-cli/releases/tag/v1.0.0). We hope that everyone upgrades and gives it a shot!

**Comment by maggie9595** [comment]
> Hi all, the fix for this issue (including @smart 's point) has been released with [v1.0.0](https://github.com/auth0/auth0-cli/releases/tag/v1.0.0). We hope that everyone upgrades and gives it a shot!

Thanks!!
